docs-src/typescript/activities.md

```diff
@@ -424,28 +432,33 @@ Activity Cancellation is an optional capability that lets you do graceful cleanu
-2. Catch a [`CancelledFailure`](/typescript/handling-failure/) while awaiting "cancellation-aware" APIs like `Context.current().sleep`. Errors can be validated with the `isCancellation(err)` utility function (see example below)
-3. Pass the context's abort Signal at [`Context.current().cancellationSignal`](https://typescript.temporal.io/api/classes/activity.Context#cancelled) to a library that supports it like `fetch`
+1. Await on the promise returned by [`cancelled()`](https://typescript.temporal.io/api/namespaces/activity#cancelled);
+2. Catch a [`CancelledFailure`](/typescript/handling-failure/) while awaiting "cancellation-aware" APIs like `sleep()`. Errors can be validated with the [`isCancellation(err)`](https://typescript.temporal.io/api/namespaces/workflow#iscancellation) utility function (see example below);
+3. Pass the context's abort Signal returned by [`cancellationSignal()`](https://typescript.temporal.io/api/namespaces/activity#cancellationsignal) to a library that supports it, like `fetch`.
 
-[`heartbeat()`](https://typescript.temporal.io/api/classes/activity.Context/#heartbeat) in the TypeScript SDK is a background operation and does not propagate errors to the caller, such as when the scheduling Workflow has already completed or the Activity has been closed by the Server (due to timeout for instance). These errors are translated into cancellation and can be handled using the methods above.
+[`heartbeat()`](https://typescript.temporal.io/api/namespaces/activity#heartbeat) in the TypeScript SDK is a background operation and does not propagate errors to the caller, such as when the scheduling Workflow has already completed or the Activity has been closed by the Server (due to timeout for instance). These errors are translated into cancellation and can be handled using the methods above.
 
 #### Example: Activity that fakes progress and can be cancelled
 
-The [`sleep`](https://typescript.temporal.io/api/classes/activity.Context#sleep) method exposed in `Context.current()` is comparable to a standard `sleep` function: `new Promise(resolve => setTimeout(resolve, sleepMS));` except that it also rejects if the Activity is cancelled.
+The [`sleep()`](https://typescript.temporal.io/api/namespaces/activity#sleep) function exported by the `@temporalio/activity` package is comparable to a standard `sleep` function: `new Promise(resolve => setTimeout(resolve, sleepMS));` except that it also rejects if the Activity is cancelled.
 
 <!--SNIPSTART typescript-activity-fake-progress-->
 
 [activities-cancellation-heartbeating/src/activities.ts](https://github.com/temporalio/samples-typescript/blob/master/activities-cancellation-heartbeating/src/activities.ts)
 
 ```ts
-import { CancelledFailure, Context } from '@temporalio/activity';
+import {
+  activityInfo,
+  CancelledFailure,
+  heartbeat,
+  log,
+  sleep,
+} from '@temporalio/activity';
```

Comment:
I would rather use `import * as activity` and not import the individual methods. it's mostly a stylistic preference but I think it ends up more readable if methods are prefixed with `activity`.